File: PCMC/Entities/Instructor.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace PCMC.Entities
{
    public class Instructor
    {
        public int ID { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public string Phone { get; set; }
    }
}

File: PCMC/Client/Common/POCO/Instructor.js
/*
 *  An Instructor class used to retain information sent from the server
 */
var Instructor = (function () {
    function Instructor(id, name, email, phone) {
        this.ID = id;
        this.Name = name;
        this.Email = email;
        this.Phone = phone;
    }
    return Instructor;
}());
